packages/experience/src/Providers/UserInteractionContextProvider/index.tsx

```diff
@@ -76,6 +79,18 @@ const UserInteractionContextProvider = ({ children }: Props) => {
     [ssoConnectors]
   );
 
+  const getIdentifierInputValueByTypes = useCallback(
+    (enabledTypes: IdentifierInputType[]) => {
+      const { type } = identifierInputValue ?? {};
+      /**
+       * Check if the type is included in the enabledTypes array
+       * If it is, return identifierInputValue; otherwise, return undefined
+       */
```

Comment:
nit: could add it as the function jsdoc comments

Reply:
Already added.
<img width="631" alt="image" src="https://github.com/user-attachments/assets/dbc4241b-4653-43bb-aad1-e6519d4827c2">